packages/core/template/portfolio/contents/project2.md

```diff
@@ -0,0 +1,23 @@
+<frontmatter>
+  title: Project 2
+</frontmatter>
+
+<br>
+
+### Project 2
```

Comment:
Feels superfluous to have this title if all the content included is in the card! Having this also makes searching pretty tricky... Just type project into the search bar and you'll see what I mean. The duplication of titles on search is outside the scope of this PR though